frontend/src/components/forms/CreateLog.tsx

```diff
@@ -66,8 +93,12 @@ function getBorderStyle(state: any, error: boolean): string {
   return "1px solid #cbd5e0";
 }
 
-const CreateLog = () => {
-  const [employee, setEmployee] = useState("Huseyin"); // currently, the select for employees is locked and should default to current user. Need to check if admins/regular staff are allowed to change this
+const CreateLog = ({
+  getRecords,
+  setUserPageNum
+}: Props) => {
+  // currently, the select for employees is locked and should default to current user. Need to check if admins/regular staff are allowed to change this
+  const [employee, setEmployee] = useState<NewSelectOptionType>({label: "", value: -1}); // currently, the select for employees is locked and should default to current user. Need to check if admins/regular staff are allowed to change this
```

Comment:
Doesn't this only need to have a value? Also instead of setting the state in a useEffect I think this would be cleaner alternative
https://legacy.reactjs.org/docs/hooks-reference.html#lazy-initial-state

Edit: Nvm, I see why you did this for the current user. Personally think keeping it as a separate state would be ideal. Also, couldn't we let `defaultValue` be a function that gets the value in local storage?